config/crd/bases/infrastructure.cluster.x-k8s.io_nutanixmachinetemplates.yaml

```diff
@@ -403,6 +403,29 @@ spec:
                         required:
                         - type
                         type: object
+                      gpus:
+                        description: List of GPU devices that need to be added to
+                          the machines.
+                        items:
+                          properties:
+                            deviceId:
```

Comment:
can we add a sample yaml in tests with data populated for deviceId. also how does a user gets this id to pass? nvidia smi? or some acli/ncli or ui?